src/functions/html.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+import defineFunction, {ordargument} from "../defineFunction";
+import buildCommon from "../buildCommon";
+import ParseNode, {assertNodeType} from "../ParseNode";
+
+import * as html from "../buildHTML";
+import * as mml from "../buildMathML";
+
+defineFunction({
+    type: "html",
+    names: ["\\class", "\\cssId"],
+    props: {
+        numArgs: 2,
+        argTypes: ["string", "original"],
```

Comment:
I'm assuming we want this allowed in text?

Reply:
@rrandallcainc Set `allowedInText` `true`.

Reply:
Assuming you'll want to apply allowInText: true also to html, no?